client/store/single-product.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+
+/**
+ * ACTION TYPES
+ */
+const GET_SINGLE_PRODUCT = 'GET_SINGLE_PRODUCT'
+
+/**
+ * INITIAL STATE
+ */
+const singleProduct = {}
+
+/**
+ * ACTION CREATORS
+ */
+const getSingleProduct = product => ({type: GET_SINGLE_PRODUCT, product})
+
+/**
+ * THUNK CREATORS
+ */
+
+export const fetchSingleProduct = productId => async dispatch => {
```

Comment:
Should we try and make naming convention for action creators and thunk creators consistent? I.e. we had something like gotItem for action creator and getItem for thunks.